python/samples/concepts/prompt_templates/configuring_prompts.py

```diff
@@ -8,61 +8,76 @@
 from semantic_kernel.functions import KernelArguments
 from semantic_kernel.prompt_template import InputVariable, PromptTemplateConfig
 
+kernel = Kernel()
 
-async def main():
-    kernel = Kernel()
+useAzureOpenAI = False
+model = "gpt-35-turbo" if useAzureOpenAI else "gpt-3.5-turbo-1106"
+service_id = model
 
-    useAzureOpenAI = False
-    model = "gpt-35-turbo" if useAzureOpenAI else "gpt-3.5-turbo-1106"
```

Comment:
why point this at 1105, there is a newer one, or just point to latest (by using gpt-3.5-turbo): https://platform.openai.com/docs/models/gpt-3-5-turbo

Reply:
Will update, good catch

Reply:
Updated all occurrences of `gpt-3.5-turbo-1106` to `gpt-3.5-turbo` which is `gpt-3.5-turbo-0125`.